src/compiler/commandLineParser.ts

```diff
@@ -7,6 +7,39 @@
 namespace ts {
     /* @internal */
     export const compileOnSaveCommandLineOption: CommandLineOption = { name: "compileOnSave", type: "boolean" };
+
+    /* @internal */
+    export const libMap = createMapFromTemplate({

```

Comment:
It would still be nice to generate this instead of writing it explicitly. Start with `const libs = ["es5", ..., "esnext.asynciterable"]`, generate the map with `lib.${x}.d.ts`, and you'll no longer need to use `arrayFrom(libMap.keys())`.